### ISSUE TITLE
Gulp: Cannot find module from /dist using browserify

### ISSUE THREAD

**Issue by daviestar** [alex3165/react-mapbox-gl#30]
Hey, trying to get 'hello world' from this package with gulp/browserify.  Facing 100s of errors like:

```
gulp-notify: [Gulp Error] Cannot find module './buffer' from '/xxx/node_modules/react-mapbox-gl/node_modules/mapbox-gl/dist'
gulp-notify: [Gulp Error] Cannot find module '../util/util' from '/xxx/node_modules/react-mapbox-gl/node_modules/mapbox-gl/dist'
gulp-notify: [Gulp Error] Cannot find module '../util/struct_array' from '/xxx/node_modules/react-mapbox-gl/node_modules/mapbox-gl/dist'
```

index.js:
```javascript
import React, { Component } from "react";
import ReactMapboxGl from "react-mapbox-gl";
import {render} from "react-dom";

const API_TOKEN = 'xxx';
const MAP_STYLE = 'mapbox://styles/mapbox/streets-v8';

export default class LondonCycle extends Component {

  render() {
    return (
      <ReactMapboxGl
        style={MAP_STYLE}
        accessToken={API_TOKEN}>
      </ReactMapboxGl>
    )
  }
}

render(<LondonCycle/>, document.getElementById('app'));
```
gulp task:
```javascript
var b = browserify({
  entries: [srcRoot + '/index.js'],
  debug: true,
  extensions: ['.jsx'],
  transform: [
    ['babelify', {presets: ['es2015', 'stage-0', 'react']}]
  ]
});
gulp.task('js', function() {
  return b.bundle()
    .on('error', handleError)
    .pipe(source('bundle.js'))
    .pipe(buffer())
    .pipe(gulp.dest(destRoot));
});
```

**Comment by alex3165** [comment]
Hello @daviestar Sorry for the late reply, to be honest i have never tried to use react-mapbox-gl with browserify, i will investigate on it as soon as possible if you still have the issue. It can be because i am using the compiled version of mapbox-gl-js, but using the source files would not work with webpack.

**Comment by daviestar** [comment]
@alex3165 after i posted this issue i discovered why you'd done it that way for webpack.  i've just copied your `src` directory into my project and removed `/dist` from the imports and it works great.

almost everyone using react seems to use webpack, but it might be worth mentioning in the docs.

**Comment by rlski** [comment]
Actually I've been using gulp and browerify for a lot of react projects until recently. If I have time i'll try to clarify the problem with browserify and make a PR for the gulp setup.

**Comment by alex3165** [comment]
If anyone is interested to work on it, it would be appreciated, i am not really used to browserify but otherwise i can try something 👍 .

**Comment by philpl** [comment]
This should be fixed with the PR #43.

Please open a new issue if any problems persist. Cheers
